test/fixtures/3.0/ping/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "ping",
+  "version": "1.0.0",
+  "description": "ping",
+  "keywords": [
+    "loopback-application",
+    "loopback"
+  ],
+  "main": "index.js",
+  "engines": {
+    "node": ">=8.9"
+  },
+  "scripts": {
+    "build:apidocs": "lb-apidocs",
+    "build": "lb-tsc es2017 --outDir dist",
+    "build:watch": "lb-tsc --watch",
+    "clean": "lb-clean dist",
+    "lint": "npm run prettier:check && npm run tslint",
+    "lint:fix": "npm run tslint:fix && npm run prettier:fix",
+    "prettier:cli": "lb-prettier \"**/*.ts\" \"**/*.js\"",
+    "prettier:check": "npm run prettier:cli -- -l",
+    "prettier:fix": "npm run prettier:cli -- --write",
+    "tslint": "lb-tslint",
+    "tslint:fix": "npm run tslint -- --fix",
+    "pretest": "npm run clean && npm run build",
```

Comment:
You need to figure out how to run the build as part of the connector's pretest step.

Personally, I would not bother with this complexity and try to write the LB4 application using JavaScript only. Since your controller provides two methods only (ping and greet), it should be reasonably easy to rewrite the handlers from controller-method style to route-handler style.

```js
app.route(verb, path, spec, handlerFn);
```